src/Javis.jl

```diff
@@ -1,5 +1,6 @@
 module Javis
 
+using Base: kwarg_decl
```

Comment:
Hi @Wikunia I don't know what `kwarg_decl` does but I think it is not called anywhere else so maybe it should not be there?

Reply:
Oh I somehow missed your review. Yes I don't know why but when I type some weird stuff in VSCode by accident some automatism adds those Imports. Will remove it.